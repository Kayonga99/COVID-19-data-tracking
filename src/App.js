import { Route, BrowserRouter as Router, Routes } from 'react-router-dom';
import Navbar from './components/Navbar';
import Home from './components/Home';
import CountryData from './components/CountryData';

function App() {
  return (
    <Router>
      <Navbar />
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/country/:name" element={<CountryData />} />
      </Routes>
    </Router>
  );
}

export default App;
