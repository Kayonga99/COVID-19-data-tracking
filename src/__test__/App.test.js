import React from 'react';
import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import store from '../redux/configureStore';
import App from '../App';

describe('App component', () => {
  test('renders', () => {
    const tree = render(
      <Provider store={store}>
        <App />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
