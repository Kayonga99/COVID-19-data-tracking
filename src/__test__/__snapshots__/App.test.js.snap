// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar"
      >
        <div>
          <a
            aria-current="page"
            class="active"
            exact="true"
            href="/"
          />
        </div>
        <h2
          class="c-data"
        >
          COVID-19 DATA
        </h2>
        <div
          class="top-right"
        >
          <div
            class="top-right"
          >
            <img
              alt=""
              class="mic"
              src="mic-svgrepo-com.svg"
            />
          </div>
          <div
            class="top-right"
          >
            <img
              alt="#"
              class="set"
              src="settings-svgrepo-com.svg"
            />
          </div>
        </div>
      </nav>
      <div
        class="heading"
      >
         
        <h1>
          EUROPE COUNTRIES' COVID-19 DATA
        </h1>
         
      </div>
      <div
        class="input-box"
      >
        <input
          class="input"
          placeholder="Serch European Country ..."
          type="text"
          value=""
        />
      </div>
      <div
        class="body"
      />
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar"
    >
      <div>
        <a
          aria-current="page"
          class="active"
          exact="true"
          href="/"
        />
      </div>
      <h2
        class="c-data"
      >
        COVID-19 DATA
      </h2>
      <div
        class="top-right"
      >
        <div
          class="top-right"
        >
          <img
            alt=""
            class="mic"
            src="mic-svgrepo-com.svg"
          />
        </div>
        <div
          class="top-right"
        >
          <img
            alt="#"
            class="set"
            src="settings-svgrepo-com.svg"
          />
        </div>
      </div>
    </nav>
    <div
      class="heading"
    >
       
      <h1>
        EUROPE COUNTRIES' COVID-19 DATA
      </h1>
       
    </div>
    <div
      class="input-box"
    >
      <input
        class="input"
        placeholder="Serch European Country ..."
        type="text"
        value=""
      />
    </div>
    <div
      class="body"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
